### ISSUE TITLE
There is an (alpha but) official ISO!

### ISSUE THREAD

**Issue by MttCastelli** [intialonso/intialonso.github.io#8]
here:

[Lubuntu Next 17.10 (Artful Aardvark) Daily Build](http://cdimage.ubuntu.com/lubuntu-next/daily-live/pending/)

It boots (on both my 2 laptop, for example) and finally has NM so you can connect to your wifi... It definitively works and it's also clean: manifest with shipped packages attached. Now you can built your iso over this or simply have an alternative. Thank you anyway.
 
[artful-desktop-amd64.manifest.txt](https://github.com/intialonso/intialonso.github.io/files/1083209/artful-desktop-amd64.manifest.txt)

**Comment by intialonso** [comment]
Yeah, I already pointed to that alpha in another issue, I guess is time lo leave this project now that Lubuntu is finally switching to LxQt :)

**Comment by lynxmedeiros** [comment]
Hello,
I would like to congratulate you for the excellent work done with UbuntuQt.
The distribution is excellent, the combination LxQt + Numix was great.
And if possible continue with the project, please do not end with UbuntuQt.
Lubuntu will come with the LxQt, but it should only arrive in the Lubuntu 18.04, Lubuntu 17.10 is still in testing.
Thank you very much in advance.
Greetings.
